src/test/java/com/ohdab/classroom/service/AddClassroomServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package com.ohdab.classroom.service;
+
+import static org.assertj.core.api.Assertions.*;
+
+import com.ohdab.classroom.repository.ClassroomRepository;
+import com.ohdab.classroom.service.dto.ClassroomReqDto;
+import com.ohdab.member.repository.MemberRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {AddClassroomService.class})
+class AddClassroomServiceTest {
+
+    @Autowired private AddClassroomService addClassroomService;
+    @MockBean private MemberRepository memberRepository;
+    @MockBean private ClassroomRepository classroomRepository;
+
+    @Test
+    @DisplayName("반 정보와 선생님 id를 통해 반을 추가 테스트")
+    void 반추가() throws Exception {
+        // given
+        Long id = 1L;
```

Comment:
```suggestion
        long id = 1L;
```
널값이 아니라는게 보장돼서 원시타입으로 선언해도 상관없어

Reply:
여기말고 다른 코드에서도 원시 타입으로 작성하는게 나을거 같애(`Long` -> `long`)